microshift_cli_ref/microshift-usage-oc-kubectl.adoc

```diff
@@ -8,6 +8,15 @@ toc::[]
 
 The Kubernetes command-line interface (CLI), `kubectl`, can be used to run commands against a Kubernetes cluster. Because {product-title} is a certified Kubernetes distribution, you can use the supported `kubectl` binaries that ship with {product-title}, or you can gain extended functionality by using the `oc` binary.
 
+[id="microshift-kubectl-binary_{context}"]
+== The kubectl binary
+
+The `kubectl` binary is provided as a means to support existing workflows and scripts for new {product-title} users coming from a standard Kubernetes environment, or for those who prefer to use the `kubectl` CLI. Existing users of `kubectl` can continue to use the binary to interact with Kubernetes primitives, with no changes required to the {product-title} cluster.
```

Comment:
I don't think we want to say "standard kubernetes" and imply that OpenShift or MicroShift are somehow not "standard".

What do the OCP docs say about why kubectl is included in the package with oc?

Reply:
This is straight from the OCP docs with no changes.

Reply:
Ugh. Keep it for now, but we should look for better wording to replace both copies.

Reply:
We could just say "another" Kube environment